pkg/agent/node-resource-topology.go

```diff
@@ -67,24 +67,13 @@ func (a *Agent) updateNrtCR(policy string, zones []*policyapi.TopologyZone) erro
 		}
 	}
 
-	// delete existing CR if we got no data from policy
 	// XXX TODO Deletion should be handled differently:
 	//   1. add expiration timestamp to nrt.NodeResourceTopology
 	//   2. GC CRs that are past their expiration time (for instance by NFD)
 	//   3. make sure we refresh our CR (either here or preferably/easier
 	//      by triggering in resmgr an updateTopologyZones() during longer
 	//      periods of inactivity)
-	if len(zones) == 0 {
-		if cr != nil {
-			err := cli.Delete(ctx, a.nodeName, metav1.DeleteOptions{})
-			if err != nil && !errors.IsNotFound(err) {
-				return fmt.Errorf("failed to delete node resource topology CR: %w", err)
-			}
-		}
-		return nil
-	}
-
```

Comment:
I know that by this, we are not covering the deletion of CR when there is no zone data. But I think we won't be even creating the CR in the first place if we don't call `UpdateNrtCR()` unnecessarily (when zone data doesn't exist).

Reply:
Yes, probably not something to worry about.